docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -39,49 +46,51 @@ spec:
 
 ## Fields
 
-* **apiVersion** -- API version being used.
-* **kind** -- Resource type.
+- **apiVersion** -- API version being used.
+- **kind** -- Resource type.
    Must be set to `KeptnApp`
 
-* **metadata**
-  * **name** -- Unique name of this application.
+- **metadata**
+  - **name** -- Unique name of this application.
     Names must comply with the
     [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
     specification.
 
-* **spec**
-  * **version** -- version of the Keptn application.
+- **spec**
+  - **version** -- version of the Keptn application.
     Changing this version number causes a new execution
     of all application-level checks
-  * **revision** -- revision of a `version`.
+  - **revision** -- revision of a `version`.
     The value is an integer that can be modified
     to trigger another deployment of a `KeptnApp` of the same version.
     For example, increment this number to restart a `KeptnApp` version
     that failed to deploy, perhaps because a
-    `preDeploymentEvaluation` or `preDeploymentTask` failed.
-  * **workloads**
-    * **name** - name of this Kubernetes
+    `preDeploymentEvaluation` or `preDeploymentTask` failed
+    for reasons that may be transient.
+  - **workloads**
+    - **name** - name of this Kubernetes
       [workload](https://kubernetes.io/docs/concepts/workloads/).
       Use the same naming rules listed above for the application name.
       Provide one entry for each workload
       associated with this Keptn application.
-    * **version** -- version number for this workload.
+    - **version** -- version number for this workload.
       Changing this number causes a new execution
-      of checks for the Keptn application and the new version of the workload.
-  * **preDeploymentTasks** -- list each task to be run
-    as part of the pre-deployment stage.
-    Task names must match the value of the `name` field
+      of checks for this workload only,
+      not the entire application.
+  - **preDeploymentTasks** -- list each task or container
```

Comment:
Ooh, I screwed this up.  I saw that `container-def` in the example but all they need to provide in KeptnApp is the name of the KeptnTaskDefinition resource, isn't it?  And maybe a comment in the Field description that this could be either a Deno task or a container -- KeptnApp doesn't care.

Reply:
yes :)

Reply:
It's fixed